cpp/daal/src/algorithms/service_kernel_math.h

```diff
@@ -396,33 +396,7 @@ class MinkowskiDistances : public PairwiseDistances<FPType, cpu>
     }
 
 protected:
-    FPType computeDistance(const FPType * x, const FPType * y, const size_t n)
-    {
-        daal::internal::mkl::MklMath<FPType, cpu> math;
-
-        FPType d = 0;
-
-        if (_p == 1.0)
-        {
-            for (size_t i = 0; i < n; ++i)
-            {
-                d += math.sFabs(x[i] - y[i]);
-            }
-
-            return d;
-        }
-        else
-        {
-            for (size_t i = 0; i < n; ++i)
-            {
-                d += math.sPowx(math.sFabs(x[i] - y[i]), _p);
-            }
-
-            if (!_powered) return math.sPowx(d, 1.0 / _p);
-
-            return d;
-        }
-    }
+    inline FPType computeDistance(const FPType * x, const FPType * y, const size_t n) const;
```

Comment:
Why are the `inline`s necessary. Just to fix comile errors?

Reply:
yes